docs/lib/analytics/fragments/android/getting-started/20_installLib.md

```diff
@@ -0,0 +1,22 @@
+Open your project `build.gradle` and add the following:
```

Comment:
Kill this stuff -- that's already done in the project setup.

Reply:
Deleted